0055-rrna/rrna.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+. ../common.sh
+
+case $# in
+    0) task=$(taskName);;
+    1) task=$1;;
+    *) echo "  $(basename $0): called with args '$@'" >&2
+       echo "  Usage: $(basename $0) [task]" >&2
+       exit 1;;
+esac
+
+task2=$(mateFile $task)
+
+fastq=../005-trim/$task.trim.fastq.gz
+fastq2=../005-trim/$task2.trim.fastq.gz
+singletons=../005-trim/$task.singletons.fastq.gz
+trimmed=$task.trimmed.fastq.gz
+
+log=$logDir/$task.log
+
+bwaDatabaseRoot="$root/share/bwa-indices"
+bwaDatabaseNames="45srRNA"
+outUncompressed=$task.rrna.out
+out=$outUncompressed.gz
+
+logStepStart $log
+logTaskToSlurmOutput $task $log
+checkFastq $fastq $log
+
+function skip()
+{
+    # Copy our input FASTQ to our output unchanged.
+    cp $fastq $out
+}
+
+# Could delete the loops - there is only one database here!
+function rrna()
+{
+	local sam=$task.sam
+	local bam=$task.bam
```

Comment:
Two extra spaces of indent here for some reason?

Reply:
hm, this was a tab including 4 spaces, so should have been the same (and appears same in sublime text). Changed all indentation to spaces.